customs.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+cd "$(realpath "$(dirname "$0")")" &&
+source bindle/project.sh
+if [ $? -ne 0 ];then
+	exit 1
+fi
+
+## deps:
+## Installs all required dependencies for Clojure and ClojureScript
+deps () {
+	#shellcheck disable=2215
+	-deps "$@"
+}
+
+## docs:
+## Generate api documentation
+docs () {
+	lein-docs
+}
+
+## lint:
+lint () {
+	#shellcheck disable=2215
+	-lint
+	docs
+	require-committed docs
+}
+
+## clean:
+clean () {
+	lein-clean
+}
+
+## test:
+## args: [-r]
+test () {
+	-test-clj "$@"
```

Comment:
Surely this fails shellcheck 2215?

Reply:
I guess it's happy with it since it takes args